src/segments/owm.go

```diff
@@ -76,9 +85,28 @@ func (d *Owm) getResult() (*owmDataResponse, error) {
 
 	apikey := d.props.GetString(APIKey, ".")
 	location := d.props.GetString(Location, "De Bilt,NL")
+	latitude := d.props.GetFloat64(Latitude, 91)
+	longitude := d.props.GetFloat64(Longitude, 181)
 	units := d.props.GetString(Units, "standard")
 	httpTimeout := d.props.GetInt(properties.HTTPTimeout, properties.DefaultHTTPTimeout)
-	d.URL = fmt.Sprintf("http://api.openweathermap.org/data/2.5/weather?q=%s&units=%s&appid=%s", location, units, apikey)
+
+	if latitude > 90 || latitude < -90 || longitude > 180 && longitude < -180 {
```

Comment:
this is a rather magical function, can we create a wrapper function that explains what it aims to validate?

```go
reasonHere := func() bool {
    return latitude > 90 || latitude < -90 || longitude > 180 && longitude < -180
}
```

Reply:
Alright, I have made the update and included comments to give clarity